java-client/src/main/java/org/opensearch/client/opensearch/ingest/TextEmbeddingProcessor.java

```diff
@@ -0,0 +1,214 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
Sorry was this class copied for Elasticsearch codebase? If not, we could only keep SPDX License

Reply:
yeah, probably something I copied and pasted from another processor class, fixed at: [752df7b](https://github.com/opensearch-project/opensearch-java/pull/1007/commits/752df7be6d17e8fdcb674adf61cce0efea77bd99)